hwdb.d/60-keyboard.hwdb

```diff
@@ -1678,6 +1678,10 @@ evdev:input:b0003v047FpC006*
 # evdev:atkbd:dmi:bvn*:bvr*:bd*:svnPurism*:pn*Librem13v4*:*
 #  KEYBOARD_KEY_56=backslash
 
+# Purism Librem 11 tablet. 114,115 are the volume keys
+evdev:atkbd:dmi:bvn*:bvr*:svnPurism:pnLibrem11:*
+ CHASSIS_KEYS=114 115
```

Comment:
so, hmm, so far all hwdb entries we defined so far generally contain a single value in their entries, or possibly a fixed number of values, but never an unbounded number of values.

Which makes we wonder if this shouldn't be done as instead:

```
CHASSIS_KEY_114=1
CHASSIS_CKEY_115=1 
```

instead?

(just asking, the answer might actually be "no").

Is it guaranteed that the number of entries here remains limited to just a bunch?

Reply:
If it's more than "just a bunch". I picked a list mostly due to parsing reasons on the compositor / libinput side.